src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { addComment, fetchPostInfo } from "../../store/post";
+import { getActivePostId } from "../../store/index";
+import { useSelector, useDispatch } from "react-redux";
+
+import "./NewCommentForm.scss";
+
+export const NewCommentForm: React.FC<{
+  setIsLoading: React.Dispatch<React.SetStateAction<boolean>>;
+}> = ({ setIsLoading }) => {
+  const activePostId = useSelector(getActivePostId);
+  const [userName, setUserName] = useState("");
+  const [email, setEmail] = useState("");
+  const [body, setBody] = useState("");
```

Comment:
объедини в один state
```suggestion
  const [comment, setComment] = useState({
    userName: '',
    email: '',
    body: '',
  });
```

Reply:
done